packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -38,6 +38,10 @@ export interface TooltipProps {
    * It controls the initial visibility of the Tooltip
    */
   isVisible?: boolean;
+  /**
+   * It controls whether or not the tooltip stays open when hovering on the content
+   */
+  canInteract?: boolean;
```

Comment:
Is this prop name clear?

Reply:
Uhmmm wdyt about renaming it to `closeOnMouseLeave`?

And then the default value is true, the user that does not want it to close onMouseLeave would need to pass

```jsx
<Tooltip closeOnMouseLeave={false}>
  {/** ... */}
</Tooltip
```

Reply:
Sounds good!